sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1437,6 +1437,15 @@ object SQLConf {
     .intConf
     .createWithDefault(20)
 
+  val FAST_HASH_AGGREGATE_MAX_ROWS_CAPACITY_BIT =
+    buildConf("spark.sql.fast.hash.aggregate.row.max.capacity.bit")
+      .internal()
+      .doc("Capacity for the max number of rows to be held in memory by the fast hash aggregate " +
+        "product operator (e.g: configuration 16 capacity size is 65536).")
```

Comment:
Can we describe something about this value? (e.g. `bit` not for `value`).

Can we describe about the default value?